app/containers/EpisodeAdmin/index.js

```diff
@@ -43,6 +43,16 @@ const getFieldChangeHandler = (name) => function (event) { // eslint-disable-lin
   this.setState({ [name]: event.target.value });
 };
 
+const getDigasEpisodeHandler = (name) => function (event, episodes) { // eslint-disable-line func-names
```

Comment:
I see you guys have grouped files into containers. As the app grows, you might find that it's easier to find things if you group containers and components based on their context. I.e. `app/sectiions/admin/episodes/index.jsx`. Common things could be grouped in `app/common`. Feel your way